documentation/book/con-zookeeper-communications.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// <List assemblies here, each on a new line>
+
+// Base the file name and the ID on the module title. For example:
+// * file name: assembly-cluster-operator.adoc
+// * ID: [id='encryption-{context}']
+// * Title: = Encryption
+
+// The ID is used as an anchor for linking to the module. Avoid changing it after the module has been published to ensure existing links are not broken.
+
+include::common/attributes.adoc[]
+
+[id='zookeeper-communication-{context}']
+= Zookeeper communication
+
+By deploying a `stunnel` sidecar within every Zookeeper pod, the Cluster Operator is able to provide data encryption and authentication between Zookeeper nodes in a cluster.
```

Comment:
This needs more information to be a concept.

Reply:
This is a low priority user story and can be revisited after the beta release.

Reply:
Same as previous comments.